addons/web/static/src/views/fields/badge_selection/badge_selection_field.js

```diff
@@ -85,6 +86,7 @@ export const badgeSelectionField = {
     isEmpty: (record, fieldName) => record.data[fieldName] === false,
     extractProps: (fieldInfo, dynamicInfo) => ({
```

Comment:
Suggestion to add (makes it editable in studio): 

`
                       supportedOptions: [ {
                            label: "Size",
                            name: "size",
                            type: "selection",
                            choices: [
                                { label: "Small", value: "sm" },
                                { label: "Medium", value: "md" },
                                { label: "Large", value: "lg" },
                            ],
                            default: "md",
                        }],`